web-ng/src/app/shared/styles/_variables.scss

```diff
@@ -37,4 +37,6 @@ $background: map-get($theme, background);
 :root {
   --drawing-tools-black: #202225;
   --drawing-tools-green: #3d7d40;
+  --project-list-background: #f0f1f2;
+  --white: #ffffff;
```

Comment:
Please use one of these colors if possible instead:
https://github.com/angular/components/blob/16f5f793dea3d83a6f08de14e3787ce97d82ff15/src/material/core/theming/_palette.scss#L674

Reply:
changed to some near matching colours unable to find exact colors.